### PR TITLE
api/client: use string.Title function where appropriate

### PR DESCRIPTION
**Pull request by stevvooe**
Use libraries.

### CODE REVIEW

**Comment by stevvooe** [comment]
Arguably, this should also do things like change `_-` to spaces, as well.

**Comment by aaronlehmann** [comment]
`strings.Title` will uppercase the first character of every word. Is this the desired behavior? The existing code only uppercases the first character of the string.

File: api/client/utils.go [hunk 1]
```diff
@@ -165,26 +165,14 @@ func (cli *DockerCli) ForwardAllSignals(ctx context.Context, cid string) chan os
...
 func PrettyPrint(i interface{}) string {
 	switch t := i.(type) {
 	case nil:
 		return "None"
 	case string:
-		return capitalizeFirst(t)
+		return strings.Title(t)
 	default:
-		return capitalizeFirst(fmt.Sprintf("%s", t))
+		return strings.Title(fmt.Sprintf("%s", t))
```
Comment (aaronlehmann) [inline api/client/utils.go]: The `Sprintf` could be simplified to `fmt.Sprint(t)`.

**Comment by thaJeztah** [comment]
> Is this the desired behavior? The existing code only uppercases the first character of the string.

I'm not sure either; looks like the go library doesn't have a similar method built-in, so we should check where it's used in our code base

**Comment by stevvooe** [comment]
@thaJeztah I really don't see the point of only capitalizing the first entry. If we look at the usage, it doesn't matter for these cases. In practice, neither approaches do the right thing in all cases. Imbalanced title casing looks weird, so it is better to use the stdlib function that define our own.

**Comment by thaJeztah** [comment]
@stevvooe ignore me; I must've been half sleeping and thought this was in a pkg, but it's not, so if this is the only place it's used, then LGTM

**Comment by LK4D4** [comment]
I'm not sure as well, it's applied to `node.Status.Message` for example. It'll be:
```
Node Is Currently Trying To Find New Manager
```

**Comment by stevvooe** [comment]
@LK4D4 O, ick.

I'm not a huge fan of blindly applying capitalization to output, but go ahead and close this.

**Comment by thaJeztah** [comment]
I'll be the bad guy :trollface: blame me :smile:

**Pull request closed without merging**